packages/html/src/multiselect/tests/multiselect-solid.tsx

```diff
@@ -0,0 +1,79 @@
+import { Chip, ChipAction } from '../../chip';
+import { MultiSelect, MultiSelectNormal, MultiSelectArrowButton, MultiSelectChipActionRemove } from '../../multiselect';
+
+const styles = `
+    #test-area {
+        grid-template-columns: 160px repeat(2, 300px);
+    }
+`;
+
+export default () =>(
+    <>
+        <style>{styles}</style>
+        <div id="test-area" className="k-d-grid">
+
+            <span></span>
+            <span>MultiSelect</span>
+            <span>MultiSelect RTL</span>
+
+            <div>placeholder</div>
+            <div>
+                <MultiSelectNormal placeholder="placeholder..." />
+            </div>
+            <div dir="rtl">
+                <MultiSelectNormal placeholder="placeholder..." />
+            </div>
+
+            <div>arrow button</div>
+            <div>
+                <MultiSelectArrowButton placeholder="multiSelect with arrow button" />
+            </div>
+            <div dir="rtl">
+                <MultiSelectArrowButton placeholder="multiSelect with arrow button" />
+            </div>
+
+            {[ 'normal', ...MultiSelect.states ].map((state) => (
+                <>
+                    <div>{ state }</div>
+                    <div>
+                        <MultiSelectChipActionRemove { ...{ [state]: true }}>{state}</MultiSelectChipActionRemove>
+                    </div>
+
+                    <div dir="rtl">
+                        <MultiSelectChipActionRemove { ...{ [state]: true }}>{state}</MultiSelectChipActionRemove>
+                    </div>
+                </>
+            ))}
+
+            <div>invalid + focus</div>
+            <div>
+                <MultiSelectChipActionRemove invalid focus>invalid + focus</MultiSelectChipActionRemove>
```

Comment:
This looks like a MultiSelect with a value - `MultiSelectChipActionRemove` sounds rather confusing to me in terms of what the template aims to showcase.

Reply:
I agree :) I renamed the template to _multiselect-value.tsx_ and respectively the component to **MultiSelectValue**